website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -227,7 +227,11 @@ HashiCorp collects the following utilization data as JSON payloads:
          - `entity` - The sum of tokens generated for a unique client identifier
          - `nonentity` - The sum of tokens without an entity attached
 - `metadata` - Optional product-specific metadata
-   - `billing_start` - The billing start date associated with the reporting cluster (license start date if not configured)
+   - `billing_start` - The billing start date associated with the reporting cluster (license start date if not configured).
+   <Note>
+    As of 1.15.13, 1.16.7, 1.17.3, 1.18 and later, the billing start date (license start date if not configured) will automatically rollover to the most recent year. 
+    Refer to <a href="upgrading/upgrade-to-1.15.x#auto-roll-billing-start-date">Upgrade to 1.15.x</a>, <a href="upgrading/upgrade-to-1.16.x#auto-roll-billing-start-date">Upgrade to 1.16.x</a>, <a href="upgrading/upgrade-to-1.17.x#auto-roll-billing-start-date">Upgrade to 1.17.x</a>, <a href="upgrading/upgrade-to-1.18.x#auto-roll-billing-start-date">Upgrade to 1.18.x</a>, for additional information on auto-roll billing start date.
+    </Note> 
```

Comment:
@schavis is this the preferred formatting, or do you have an alternative suggestion?

Reply:
I added a previous suggestion for the aside. Let me know if you have any questions about it :)